src/formatly.ts

```diff
@@ -56,7 +56,7 @@ export async function formatly(
 		ran: true,
 		result: await new Promise((resolve, reject) => {
 			const child = spawn(baseCommand, [...args, ...patterns], {
-				cwd: process.cwd(),
+				cwd: options.cwd ?? process.cwd(),
```

Comment:
[Testing] It'd be nice to add unit tests to ensure the right cwd is used. Do you have time to add a couple to `formatly.test.ts`?

Reply:
Sure! Added one test to `formatly.test.ts`.

And a new "integration" test file as well as I think it makes sense to have a "clean" but complete test for detecting the formatter inside a custom `options.cwd` and then spawning the right process based on that. A new file, because for this test `mockResolveFormatter` must not be mocked. Happy to cut down to test to fit it into `resolveFormatter.test.ts`, though. Just let me know what makes sense to you, I'd fully understand the idea to stick to having unit tests only.

Reply:
Yeah, I normally don't personally go for integration tests - but also haven't worked on a project quite like formatly before (cwd sensitivity, shell scripts, etc.). I think this makes sense to try out. Let's go for it! 🥳 

Also filed #142 for end-to-end tests if you're interested.